src/Evolution/DgSubcell/Actions/Initialize.hpp

```diff
@@ -105,10 +110,9 @@ struct Initialize {
       const ArrayIndex& array_index, ActionList /*meta*/,
       const ParallelComponent* const /*meta*/) noexcept {
     const SubcellOptions& subcell_options = db::get<Tags::SubcellOptions>(box);
-    // Intentional copy of mesh
-    const Mesh<Metavariables::volume_dim> dg_mesh =
+    const Mesh<Metavariables::volume_dim>& dg_mesh =
         db::get<::domain::Tags::Mesh<Metavariables::volume_dim>>(box);
-    const Mesh<Metavariables::volume_dim> subcell_mesh = fd::mesh(dg_mesh);
+    const Mesh<Metavariables::volume_dim>& subcell_mesh = fd::mesh(dg_mesh);
```

Comment:
Should this go back to a non-reference, because you're creating a new mesh for subcell, rather than pointing to an existing mesh in the `box` as above?

Reply:
yes, you're right :)